dizoo/taxi/envs/taxi_env.py

```diff
@@ -0,0 +1,192 @@
+import copy

```

Comment:
please polish the import sequence according to the doc [here](https://di-engine-docs.readthedocs.io/zh-cn/latest/21_code_style/index.html#reasonable-import-order)